src/gcp/cloudfunctionsv2.ts

```diff
@@ -18,6 +18,14 @@ const client = new Client({
 });
 
 export const PUBSUB_PUBLISH_EVENT = "google.cloud.pubsub.topic.v1.messagePublished";
+export const STORAGE_EVENTS = [
+  "google.cloud.storage.object.v1.finalized",
+  "google.cloud.storage.object.v1.archived",
+  "google.cloud.storage.object.v1.deleted",
+  "google.cloud.storage.object.v1.metadataUpdated",
+];
+
+export const PUBSUB_V2_EVENT = "google.cloud.pubsub.topic.v1.messagePublished";
```

Comment:
In `parseTriggers.ts` we're only importing `PUBSUB_PUBLISH_EVENT` from L20, should we remove this line?

Reply:
Ah good catch!